app/Models/CompanyDayLog.php

```diff
@@ -5,29 +5,6 @@
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 
-/**
- * App\Models\CompanyDayLog
- *
- * @property int $id
- * @property int $company_id
- * @property int $produced
- * @property int $demand
- * @property int $stored
- * @property \Illuminate\Support\Carbon|null $created_at
- * @property \Illuminate\Support\Carbon|null $updated_at
- * @property-read \App\Models\Company $company
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog newModelQuery()
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog newQuery()
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog query()
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog whereCompanyId($value)
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog whereCreatedAt($value)
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog whereDemand($value)
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog whereId($value)
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog whereProduced($value)
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog whereStored($value)
- * @method static \Illuminate\Database\Eloquent\Builder|CompanyDayLog whereUpdatedAt($value)
- * @mixin \Eloquent
- */
```

Comment:
Why art thou removing this?

Reply:
Let me add it back in then, dacht dat het random toegevoegd werd.